src/js/routes/Vote.jsx

```diff
@@ -419,129 +420,124 @@ class Vote extends Component {
     const electionDayTextFormatted = electionDayText ? <span>{moment(electionDayText).format('MMM Do, YYYY')}</span> : <span />;
 
     return (
-      <div id="the next styled div is the VoteContainer">
-        <VoteContainer>
-          <div className={`ballot__heading-vote-section ${ballotHeaderUnpinned && isWebApp() ? 'ballot__heading__unpinned' : ''}`} style={cordovaVoteMiniHeader()}>
-            <div className="page-content-container">
-              <div className="container-fluid">
-                <div className="row">
-                  <div className="col-md-12">
-                    <Helmet title="Vote - We Vote" />
-                    <header className="ballot__header__group">
-                      <h1 className={isCordova() ? 'ballot__header__title__cordova' : 'ballot__header__title'}>
-                        { electionName ? (
-                          <span className={isWebApp() ? 'u-push--sm' : 'ballot__header__title__cordova-text'}>
-                            {electionName}
-                            {' '}
-                            <span className="d-none d-sm-inline">&mdash; </span>
-                            <span className="u-gray-mid u-no-break">{electionDayTextFormatted}</span>
-                          </span>
-                        ) : (
-                          <span className="u-push--sm">
-                           Loading Election...
-                          </span>
-                        )}
-                      </h1>
-                    </header>
-                    <div className="ballot__filter__container">
-                      <BallotFilterRow>
-                        <div className="ballot__item-filter-tabs">
-                          { ballotWithItemsFromCompletionFilterType && ballotWithItemsFromCompletionFilterType.length ? (
-                            <React.Fragment>
-                              <BallotSearch
-                                isSearching={isSearching}
-                                onToggleSearch={this.handleToggleSearchBallot}
-                                items={ballotWithItemsFromCompletionFilterType}
-                                onBallotSearch={this.handleSearch}
-                                alwaysOpen
-                              />
-                            </React.Fragment>
-                          ) : null
-                          }
-                        </div>
-                      </BallotFilterRow>
-                    </div>
+      <VoteContainer>
```

Comment:
This is another example @lazorfuzz -- could you please add these div's back into this pull request?
div id="the next styled div is the VoteContainer"

Reply:
@DaleMcGrew sure, I can add them back. Although I don't believe they will be necessary at this point. I assumed @SailingSteve added them because styled-components was harder to debug. I added `babel-plugin-styled-components` to help with the debugging issue. With the plugin the DOM nodes generated by styled should have richer output in the class attribute: https://www.styled-components.com/docs/tooling#better-debugging

Reply:
Thank you @lazorfuzz, `babel-plugin-styled-components` sounds great! I will ask @SailingSteve to remove those divs if `babel-plugin-styled-components` solves the problem he is facing in Cordova.